skimage/data/__init__.py

```diff
@@ -259,3 +259,20 @@ def rocket():
 
     """
     return load("rocket.jpg")
+
+def _install_metadata_doc():
+    "Insert metadata to docstrings that return images."
+    import sys
+    for name in __all__:
```

Comment:
I don't think I'm a fan of this approach. What if we want to add new functions? It's up to the contributor to figure out that this global function is messing with their contribution.

Instead, I would suggest a decorator that gets applied to each image function. It's a bit more wordy, but also more self-documenting.

As an alternative, I would make this list *explicit*, instead of implicitly loading all names.